apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -965,6 +968,8 @@ export const InfiniteEventTypeList = ({
   }, []);
 
   if (!pages?.[0]?.eventTypes?.length) {
+    if (isPending) return <SkeletonLoader />;
```

Comment:
Why do we need to check it here?

Can we not early return the Skeleton from the parent `InfiniteMobileTeamsTab` (By the way it isn't mobile specific and could be renamed `InfiniteEventTypeListWrapper`) itself.

Reply:
> Can we not early return the Skeleton from the parent InfiniteMobileTeamsTab

we can't early return skeleton loader in InfiniteMobileTeamsTab because then `buttonInView` would only be visible after first page of event types page is loaded and it would execute fetchNextPage() so two pages would be loaded instead of one.  I had tried this before in my last PR.

Reply:
That's a weird reason. I feel we could find a way to fix it but not important to focus on.